src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -239,32 +264,99 @@ function ProjectInfo({ project }: Props): ReactElement {
       )}
 
       <div style={{ display: 'flex' }}>
-        {project?.metadata?.siteOwnerName && (
+        {project?.metadata?.siteOwnerType && (
           <div className={styles.projectMoreInfo}>
             <div className={styles.infoTitle}>
               {t('manageProjects:siteOwnership')}
             </div>
-            {project.metadata?.siteOwnerType && (
-              <div className={styles.infoText} style={{ fontWeight: 'bold' }}>
+            {project?.metadata?.siteOwnerType && (
+              <div className={styles.infoText}>
                 {ownerTypes.map((ownerType: any, index: any) => {
                   return (
                     <React.Fragment key={ownerType}>
                       {t(`manageProjects:${ownerType}`)}
-                      {index === ownerTypes.length - 2
-                        ? ' and '
-                        : index === ownerTypes.length - 1
-                        ? '.'
-                        : ', '}
+                      {index === ownerTypes.length - 2 ? (
+                        <> {t('manageProjects:and')} </>
+                      ) : index === ownerTypes.length - 1 ? (
+                        '.'
+                      ) : (
+                        ', '
+                      )}
                     </React.Fragment>
                   );
                 })}
               </div>
             )}
+            {project?.metadata?.siteOwnerName ||
+            project?.metadata?.acquisitionYear ? (
+              <div className={styles.infoText}>
+                {project?.metadata?.siteOwnerName}{' '}
+                {project?.metadata?.siteOwnerName &&
+                project?.metadata?.acquisitionYear ? (
+                  <> {t('manageProjects:since')}</>
```

Comment:
Not sure if this is in this line or at line 345, but there is a space missing between the word "since" and the following year.

<img width="220" alt="Bildschirmfoto 2022-06-19 um 09 55 29" src="https://user-images.githubusercontent.com/1532418/174471450-d1369296-e8b7-4df2-adcb-5d1f84908ee3.png">